es-model-latest/file_centric.mapping.json

```diff
@@ -107,35 +125,41 @@
                   "type": "integer"
                 }, 
                 "created_at": {
-                  "format": "yyyy-MM-dd HH:mm:ss.SSSSSS||yyyy-MM-dd HH:mm:ss", 
+                  "format": "yyyy-MM-dd HH:mm:ss.SSSSSS||yyyy-MM-dd HH:mm:ss||yyyy-MM-dd'T'HH:mm:ss.SSSSSS+HH:mm", 
                   "type": "date"
                 }, 
-                "diagnosis": {
-                  "type": "keyword"
-                }, 
                 "diagnosis_category": {
                   "type": "keyword"
                 }, 
                 "external_id": {
                   "type": "keyword"
                 }, 
-                "icd_id": {
+                "icd_id_diagnosis": {
                   "type": "keyword"
                 }, 
                 "kf_id": {
                   "type": "keyword"
                 }, 
                 "modified_at": {
-                  "format": "yyyy-MM-dd HH:mm:ss.SSSSSS||yyyy-MM-dd HH:mm:ss", 
+                  "format": "yyyy-MM-dd HH:mm:ss.SSSSSS||yyyy-MM-dd HH:mm:ss||yyyy-MM-dd'T'HH:mm:ss.SSSSSS+HH:mm", 
                   "type": "date"
                 }, 
-                "mondo_id": {
+                "mondo_id_diagnosis": {
+                  "type": "keyword"
+                }, 
+                "ncit_id_diagnosis": {
+                  "type": "keyword"
+                }, 
+                "source_text_diagnosis": {
                   "type": "keyword"
                 }, 
-                "tumor_location": {
+                "source_text_tumor_location": {
                   "type": "keyword"
                 }, 
-                "uberon_id": {
+                "spatial_descriptor": {
+                  "type": "keyword"
+                }, 
+                "uberon_id_tumor_location": {
```

Comment:
This is called `uberon_id_anatomical_site` in the datamodel?

Reply:
there are both `uberon_id_anatomical_site` (under `Biospecimen`) and `uberon_id_tumor_location` (under `Diagnosis`)